.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,6 +9,8 @@ Related to issue #
 [//]: # (Stay ahead of things, add list items here!)
 - [ ] Clean up commit history
 
+- [ ] Update any affected documentation
```

Comment:
**nit:** we should change this phrasing to something more generic in the case no documentation exists for the thing.

Reply:
I've thought about this and can't come up with anything reasonably succinct. "If changes require any documentation updates (in this library or another), make them" or something? I think once the template is in use for all our libs, we can just leave this line  out when merging into non-doc ones. 

But if you have preferred wording, open for recommendations!

Reply:
Haha not a big deal... maybe something like "Add or update documentation related to these changes if appropriate" ??

Reply:
After looking at the new issue templates, I'm going to make all related changes in a separate PR.